.github/workflows/build.yml

```diff
@@ -14,51 +14,27 @@ jobs:
   tests:
     strategy:
       fail-fast: false
-      max-parallel: 4
+      max-parallel: 5
       matrix:
         platform: [ubuntu-latest] #, windows-latest]
-        python-version: [3.8, 3.9, '3.10', '3.11', '3.12']
-        include:
-          - python-version: 3.8
-            tox-env: py38
-            continue-on-error: False
-          - python-version: 3.9
-            tox-env: py39
-            continue-on-error: False
-          - python-version: '3.10'
-            tox-env: py310
-            continue-on-error: False
-          - python-version: '3.11'
-            tox-env: py311
-            continue-on-error: False
-          - python-version: '3.12'
-            tox-env: py312
-            continue-on-error: False
+        python-version: [3.8, 3.9, '3.10', 3.11, 3.12]
         # exclude:
         #   - platform: windows-latest
         #     python-version: 3.9
 
-    env:
-      TOXENV: ${{ matrix.tox-env }}
-
     runs-on: ${{ matrix.platform }}
 
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - name: Set up Python ${{ matrix.python-version }}
-      if: "!endsWith(matrix.python-version, '-dev')"
```

Comment:
None of the `matrix.python-version` values end in `-dev` so this test is not needed.

Reply:
Again, this is part of deadsnakes (when we use it), but as I stated, I am okay removing this as now we can utilize prereleases.